modules/prebid/rulesengine/result_functions.go

```diff
@@ -84,19 +84,18 @@ func (eb *ExcludeBidders) Call(req *openrtb_ext.RequestWrapper, result *hs.HookR
 		//create an analytics tag
 	}*/
 
-	// maybe merge all args into one to remove for loop - in res funct constructor NewIncludeBidders
-	for _, arg := range eb.Args {
-		if arg.IfSyncedId {
-			// possibly modify args.bidders
-		}
-		// build map[impId] to map [bidder] to bidder params
-		impIdToBidders, err := buildExcludeBidders(req, arg.Bidders)
-		if err != nil {
-			return err
-		}
+	if eb.Args.IfSyncedId {
+		// possibly modify args.bidders
+	}
 
-		result.ChangeSet.BidderRequest().Bidders().Update(impIdToBidders)
+	// build map[impId] to map [bidder] to bidder params
+	impIdToBidders, err := buildExcludeBidders(req, eb.Args.Bidders)
+	if err != nil {
+		return err
 	}
+
+	result.ChangeSet.BidderRequest().Bidders().Update(impIdToBidders)
```

Comment:
I think `hooks/hookstage/bidderrequest_mutations.go#Update` needs to call `impExt.SetPrebid(impPrebid)` after `impPrebid.Bidder = impBidders`.